src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -347,6 +361,9 @@ export class InputDatePicker
   // --------------------------------------------------------------------------
 
   connectedCallback(): void {
+    this.activeHandler(this.active || this.open);
+    this.openHandler(this.active || this.open);
```

Comment:
it's better to invoke these methods only if the user sets `active || open` to `true `